csharp/DEVELOPER.md

```diff
@@ -4,30 +4,68 @@ This document describes how to set up your development environment to build and
 
 ### Development Overview
 
-The GLIDE C# wrapper consists of both C# and Rust code.
+We're excited to share that the GLIDE C# client is currently in development! However, it's important to note that this client is a work in progress and is not yet complete or fully tested. Your contributions and feedback are highly encouraged as we work towards refining and improving this implementation. Thank you for your interest and understanding as we continue to develop this C# wrapper.
+
+The C# client contains the following parts:
+
+1. Rust part of the C# client located in `lib/src`; it communicates with [GLIDE core rust library](../glide-core/README.md).
+2. C# part of the client located in `lib`; it translates Rust async API into .Net async API.
+3. Integration tests for the C# client located in `tests` directory.
+4. A dedicated benchmarking tool designed to evaluate and compare the performance of GLIDE for Redis and other .Net clients. It is located in `<repo root>/benchmarks/csharp`.
```

Comment:
```suggestion
4. A dedicated benchmarking tool designed to evaluate and compare the performance of GLIDE for Redis and other .Net clients. It is located in `benchmarks/csharp`.
```

Reply:
Could be cofusing, because I refer `lib` and `tests` which are located in `<repo root>/csharp`. There is no `benchmark` near by.